src/pa/mod.rs

```diff
@@ -706,13 +707,15 @@ impl<I: Sample, O: Sample> Stream<I, O> {
     /// * frames_per_buffer - The number of frames in the buffer.
     ///
     /// Return NoError on success, or a Error code if fail.
-    pub fn write(&self, output_buffer: Vec<O>, frames_per_buffer : u32) -> Result<(), Error> {
+    pub fn write(&self, output_buffer: Vec<O>, frames_per_buffer : u32) -> Result<Option<WriteFlags>, Error> {
```

Comment:
This line now exceeds 80 characters. Any suggestions on how to break this into two lines?